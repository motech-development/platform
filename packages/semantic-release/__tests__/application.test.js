const application = require('../application');

describe('application', () => {
  it('should have the correct config', () => {
    expect(application).toMatchSnapshot();
  });
});
