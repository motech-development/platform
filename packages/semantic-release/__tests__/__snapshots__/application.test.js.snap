// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`application should have the correct config 1`] = `
{
  "branches": [
    "main",
    "next",
  ],
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    [
      "@semantic-release/npm",
      {
        "npmPublish": false,
      },
    ],
    [
      "@semantic-release/github",
      {
        "failComment": false,
        "labels": false,
        "successComment": false,
      },
    ],
  ],
}
`;
